.condarc

```diff
@@ -0,0 +1,5 @@
+channel_priority: strict
+channels:
+  - birdhouse
```

Comment:
We probably do not need birdhouse channel anymore.  For which package do we need it?